sig-contributor-experience/triage-team/triage.md

```diff
@@ -0,0 +1,69 @@
+
+The SIG Contributor Experience [issues](https://github.com/kubernetes/community/labels/sig%2Fcontributor-experience) and [PRs](https://github.com/kubernetes/community/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+label%3Asig%2Fcontributor-experience+) are labeled with `sig/contributor-experience`.
+
+Following are few example searches on issue and PR:
+* [Open issues](https://github.com/kubernetes/community/labels/sig%2Fcontributor-experience)
+* [Open issues for a specific milestone](https://github.com/kubernetes/community/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3Asig%2Fcontributor-experience+milestone%3AMay)
+* [Open PRs](https://github.com/kubernetes/community/pulls?utf8=%E2%9C%93&q=is%3A+pr+is%3Aopen+label%3Asig%2Fcontributor-experience+)
+* [Open PRs for a specific milestone](https://github.com/kubernetes/community/pulls?utf8=%E2%9C%93&q=is%3Apr+label%3Asig%2Fcontributor-experience+milestone%3AMay+is%3Aopen)
+
+## Scope
+
+Manage incoming issues and PRs that are already identified and labeled with the `sig/contributor-experience` particularly in the [Kubernetes community repository](https://git.k8s.io/community) but in general across various Kubernetes related GitHub repositories like [Kubernetes repository](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Asig%2Fcontributor-experience).
+
+## People involved
+
+Everyone is welcome to help manage issues and PRs but the work and responsibilities discussed in this document is created with Triage team and Triage Captain roles in mind.
+
+## Work involved
+
+The main work is to keep track of issues and PRs, ping people, add appropriate labels as needed and report the overall status to the SIG Leads.
+
+  - Validate if an issue is a bug - Validate if the issue is indeed a bug by following these [instructions](https://git.k8s.io/community/contributors/guide/issue-triage.md#validate-if-the-issue-is-bug).
+
+  - Determine it belongs to contributor-experience SIG - An issue can be incorrectly labeled with the `sig/contributor-experience` label. If you think an issue does not belong to the `sig/contributor-experience`, add a comment stating so with a reason. Example response:
+
+```code
+
+`/remove-sig contributor-experience`
+
+I don't think this is a SIG Contributor Experience issue because ....
+
+If you think this is incorrect, please provide a reason and add it back with /sig contributor-experience label as a single line comment.
+
+```
+
+  - Define priorities - Define priority of an issue or PR by following these [instructions](https://git.k8s.io/community/contributors/guide/issue-triage.md#define-priority).
+
+  - Verify important tasks and labels are in place - Make sure that one or more of the following tasks and labels are identified for issues and PRs. If any of these are missing, you can add one if you are authorized to do so. If you can not assign or can not decide the correct one, that’s fine, contact leaders of the SIG to do so.
```

Comment:
I am open make any changes to this list if you have any suggestion. Thanks!

Reply:
This checklist is lacking context or instructions that would explain what I'm supposed to do.  eg:
- For Issue, does "good first issue" mean I should ensure every issue has that label?
- For PR, what does "Milestone" mean?

Reply:
@spiffxp hi Aaron, I see your point and agree with the confusion that could create to the users of these guidelines. My assumption was that the triage team has understanding of these labels, and also that folks will look at the usage/explanation of the labels to get familiar with them. As you noted, definitely not all issues are "good first issue".. How about I add few lines to suggest users to look these links to understand the meaning of labels and commands https://prow.k8s.io/command-help and https://github.com/clandry94/community/blob/6d3c337df3ea6f71cc5a536523a25c8afbc7d7d2/contributors/guide/README.md#find-a-good-first-topic or any better links that you can think of? Thanks!!

Reply:
I would not make that assumption, and instead err on the side of being explicit, linking to docs for context, and providing examples.  It would probably be educational if you asked for volunteers for this team and used this doc to onboard them.

For the milestone thing, for example, since we're being specific to sig contribex, you can't just link to how the release team uses milestones.  How does sig contribex use milestones?  When should I put something in the current milestone vs. some other milestone? How do I know what the current milestone is?

Reply:
@spiffxp OK, thanks Aaron and yes that makes sense. I will be more explicit. @parispittman @nikhita @cblecker do we have milestone and or other labels that are defined specifically in how contribEx uses them? Also, other things as Aaron mentioned -  any docs that we have on  criteria for setting or changing milestone? If we already have something, I can link it/reuse. If not, I will create something and send it for review. Thanks!

Reply:
@parispittman @nikhita @cblecker  hello, a gentle reminder ping about my question here :-) Thanks!